src/data/queries/index.ts

```diff
@@ -100,20 +105,53 @@ export const QUERIES_MAP = {
   'vamc-ehr': VamcEhr,
 }
 
-// Type representing all possible object shapes returned from querying and formatting Drupal data.
-// E.g. StoryListingType | NewsStoryType | (other future resource type)
-// E.g. Accordion | Button (other paragraphs)
-// Type constructed by:
-//  1. Consider all ResourceType or ParagraphResourceType types
-//  2. Take subset of those types that have a key in QUERIES_MAP
-//  3. Map that subset of keys to their respective values, which are modules for querying data
-//  4. Within each of those modules, grab the return type of the `formatter` function
+// All resource types that have a `data` function defined
```

Comment:
I really like this splitting up & individual comments for everything. A little easier to follow.